docs/accessing-data/overview.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Overview"
+sidebar_position: 0
+---
+
+## What is Hubble?
+
+Hubble is an open-source, publicly available dataset that provides a complete historical record of the Stellar network. Similar to Horizon, it ingests and presents the data produced by the Stellar network in a format that is easier to consume than the performance-oriented data representations used by Stellar Core. The dataset is hosted on BigQuery–meaning it is suitable for large, analytic workloads, historical data retrieval and complex data aggregation. Hubble should not be used for real-time data retrieval and cannot submit transactions to the network.
```

Comment:
Should we bold the last line?
`Hubble should not be used for real-time data retrieval and cannot submit transactions to the network.`